enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -4252,8 +4252,8 @@ class AdjointGenerator
     SmallVector<Value *, 8> pre_args = {0, 0, 0};
     std::vector<DIFFE_TYPE> argsInverted = {DIFFE_TYPE::CONSTANT,
```

Comment:
Change this vector too?

Reply:
done

Reply:
turns out it is really annoying to have a `SmallVector` instance inside the cache keys.